README.md

```diff
@@ -31,14 +31,20 @@ If you'd like to hack on Scilla, clone it with all of its submodules:
 git clone --jobs 4 --recurse-submodules https://github.com/Zilliqa/scilla/
 ```
 
-### Build requirements
+### 2. Build prerequisites
 
 Platform specific instructions for setting up your system for building Scilla can be
 found in [INSTALL.md](./INSTALL.md).
 
-### Compiling and Running
+### 3. Compiling
 
-To build the project, run `make clean; make` from the root folder.
+To build the project from the root folder:
```

Comment:
Just the build part `make` should still be there. It builds in the project folder itself.

`make install` installs Scilla to your local opam switch. So that should be a separate section "Install".

Reply:
I tried `make`, but it didn't work.
```
 ✘ snow@Hans-MacBook-Pro  ~/Desktop/testing_folder/scilla   master  ./scilla-checker -gaslimit 10000 -libdir src/stdlib tests/contracts/auction.scilla
zsh: no such file or directory: ./scilla-checker
```

Was suggested to try either `make dev` for local use, or `make install`. Both works for me.

Reply:
> I tried `make`, but it didn't work.
> 
> ```
>  ✘ snow@Hans-MacBook-Pro  ~/Desktop/testing_folder/scilla   master  ./scilla-checker -gaslimit 10000 -libdir src/stdlib tests/contracts/auction.scilla
> zsh: no such file or directory: ./scilla-checker
> ```
> 
> Was suggested to try either `make dev` for local use, or `make install`. Both works for me.

@evesnow91 With https://github.com/Zilliqa/scilla/pull/792 being merged, you should *now* be able to see it with `make` also.